docs/blockchain/contracts/nft-contract/nft-actions/stofrcfg.a.experimental.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'stofrcfg.a'
+order: 36
+
+---
+
+# stofrcfg.a
+
+Set global Uniq offer configuration
+
+## Behavior
+
+Set global configurations for all Uniq offers that will be made on either a Uniq or a Uniq factory.
+
+## Technical Behavior
+
+Only `ultra.nft.ft` account can set buy offer config
+
+The action stores the configurations to `offercfg.a` singleton table with the specified arguments.
+
+All fields are optional, the action will only update the specified arguments and leave the rest the same as existing entry or as the default value if there’s no existing entry.
+
+The action even accepts the same values as the ones currently stored in `offercfg.a`.
+
+`eosio.nft.ft` pays RAM usage.
+
+`min_price` must be positive, and only support UOS and USD.
+
+Both `min_duration` and `max_duration` must be positive, with `max_duration` must be greater than `min_duration`.
+
+`max_active_offer_per_user` must also be positive.
+
+## Action Parameters
+
+| Property Name             | C++ Type            | JavaScript Type    | Definition                                                                    |
+| ------------------------- | ------------------- | ------------------ | ----------------------------------------------------------------------------- |
+| min_price                 | optional\<asset>    | String/Null        | Minimum allowed offer price                                                   |
```

Comment:
For this and below rows, "allowed" can be omitted and still makes sense.

Reply:
updated